### PR TITLE
Fixed default swift version

### PR DESCRIPTION
**Pull request by gtrop1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ All notable changes to this project will be documented in this file.
 #### Removed
 
 #### Fixed
+- Default Swift version now correctly set to 4.2.
```
Comment (cnoon) [inline CHANGELOG.md]: How about: `- Issue where Swift version was incorrectly set to 5.0 when it should have been 4.2.`?

**Review by cnoon — CHANGES_REQUESTED** [review]
Let's just wrap all the Info.plist and podspec changes into this as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -43,6 +43,14 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [5.2.1](https://github.com/Nike-Inc/Willow/releases/tag/5.2.1)
+
+#### Fixed
+- Issue where Swift version was incorrectly set to 5.0 when it should have been 4.2..
```
Comment (cnoon) [inline CHANGELOG.md]: Extra `.` at the end.

**Review by cnoon — CHANGES_REQUESTED** [review]
One small comment back.

**Review by dfuerle — APPROVED** [review]

**Review by cnoon — APPROVED** [review]
Looks good!

**Pull request merged**